include/os_portable.h

```diff
@@ -74,6 +74,9 @@
 #ifndef PORTABLE_H
 #define PORTABLE_H
 
+#include <stddef.h>

```

Comment:
This is actually code provided by TI, and changing it here will cause halcogen to overwrite it next time someone runs it and these changes may get squashed

Reply:
I just removed the changes to os_portable.h